main.go

```diff
@@ -175,7 +175,7 @@ func main() {
 	githubClient := createGitHubClient(conf)
 	r := gin.Default()
 
-	r.POST("/incoming", func(context *gin.Context) {
+	r.POST("/", func(context *gin.Context) {
```

Comment:
Currently as workaround I configured nginx reverse proxy to replay 200 on GET to `/` endpoint. This is needed for google load balancer to determinate if service is healthy. If LB doesn't get 200 replies then it doesn't forward traffic to the container. Would be great to have this as a part of test runner bot itself.

Reply:
I don't understand: do you want me to revert this change?

I know that you configured nginx to handle this, but I think we can now remove that handling and just let the container reply

Reply:
Now I understood, thanks. Fixing...

Reply:
Done :+1: